testing/start.rst

```diff
@@ -0,0 +1,64 @@
+How to test a Plone product
+===========================
+
+To better understand how to test a Plone product, the best thing is to create a new Plone package from scratch
+and use it to see different testing techniques.
+
+We are going to use `plonecli <https://pypi.org/project/plonecli/>`_ tool heavily because it allows to easily create new Plone package
+and features (content-types, views, vocabularies) with simple commands.
+
+Create Package
+--------------
+
+First of all we need to create a virtual Python environment:
+
+.. code-block:: console
+
+  $ virtualenv .env
+
+
+.. note::
+
+  By default you should use Python3 and its virtualenv because Python2 will be deprecated on 01/01/2020.
+
+Activate virtual Python environment:
+
+.. code-block:: console
+
+  $ source .env/bin/activate
+
+Install plonecli:
+
+.. code-block:: console
+
+  $ pip install plonecli
+
+Create a new addon package where you want:
+
+.. code-block:: console
+
+  $ plonecli create addon plonetraining.testing
+
+
+Buildout
+--------
+
+Run buildout:
+
+.. code-block:: console
+
+  $ cd plonetraining.testing
+  $ pip install -r requirements.txt
+  $ bin/buildout
```

Comment:
Does this work? If one does `pip install` something in an activated Virtualenv then the executable is installed in the `.env/bin` AFAIK. So, the call should be just `buildout`, since `.env/bin` its in the `PATH`, right? Correct me if I am wrong. I am using pyenv for a while now and maybe virtualenv changed since.

Reply:
You're right. It shouldn't work.
I use pipenv and i copied this snippet from another training.

Probably the best approach is using plonecli documentation for installation as you suggested in the other comment.  Thank you

Reply:
fixed installation part